navigation/mapping_exploration/launch/explore_lite.launch

```diff
@@ -0,0 +1,16 @@
+<launch>
+<node pkg="explore_lite" type="explore" respawn="true" name="explore" output="screen">
```

Comment:
The namespace should be set here and removed it from the topic parameter.

```suggestion
<node pkg="explore_lite" type="explore" respawn="true" name="explore" output="screen" ns="$(arg ns)">
```

Reply:
Done!